docs/scenes/get-started/start-developing/TemplateFeatures/features/themes.mdx

```diff
@@ -0,0 +1,5 @@
+import { RawH3 } from '~/ui/components/Text';
+
+<RawH3>Light and dark modes</RawH3>
+
+This template has light and dark mode support. The `useColorScheme()` hook lets you inspect what the user's current color scheme is, so that you can adjust UI colors accordingly.
```

Comment:
```suggestion
This template has light and dark mode support. The `useColorScheme()` hook lets you inspect the user's current color scheme so that you can adjust UI colors accordingly.
```